main.go

```diff
@@ -81,6 +81,8 @@ var (
 	metricsPath      = kingpin.Flag("metrics_path", "Path under which to expose metrics for the bridge ($METRICS_PATH)").Envar("METRICS_PATH").Default("/metrics").String()
 	extendedDetails  = kingpin.Flag("extended_details", "When enabled, alerts are presented in HTML format and include colorized status (FIR|RES), alert start time, and a link to the generator of the alert ($EXTENDED_DETAILS)").Default("false").Envar("EXTENDED_DETAILS").Bool()
 	dispatchErrors   = kingpin.Flag("dispatch_errors", "When enabled, alerts will be tried to dispatch with a error-message regarding faulty templating or missing fields to help debugging ($DISPATCH_ERRORS)").Default("false").Envar("DISPATCH_ERRORS").Bool()
+	markdown         = kingpin.Flag("markdown", "Renders the templates as Markdown ($MARKDOWN)").Default("false").Envar("MARKDOWN").Bool()
+	clickToGenerator = kingpin.Flag("click_to_generator", "Makes the notification clickable, leading to the generator URL ($CLICK_TO_GENERATOR)").Default("false").Envar("CLICK_TO_GENERATOR").Bool()
```

Comment:
I haven't tested, but is the generatorUrl always set? I've come to learn that there are a few flavors of alertmanagers out there (both the Prometheus and Grafana projects have ones that subtly differ). I ask because I'm wondering if this help line should add an "if generatorUrl is present in the alert" to the end.

Reply:
I will fix that and add the `generatorURL` check!